plugins/source/googleanalytics/client/report.go

```diff
@@ -0,0 +1,80 @@
+package client
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cloudquery/plugin-sdk/caser"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	analyticsdata "google.golang.org/api/analyticsdata/v1beta"
+)
+
+type Report struct {
+	Name          string    `json:"name"`
+	Dimensions    []string  `json:"dimensions,omitempty"`
+	Metrics       []*Metric `json:"metrics"`
+	KeepEmptyRows bool      `json:"keep_empty_rows,omitempty"`
+}
+
+var csr = caser.New()
+
+func (r *Report) normalize() {
+	if r == nil {
+		return
+	}
+
+	r.Name = csr.ToSnake(strings.ReplaceAll(r.Name, " ", "_"))
```

Comment:
Shouldn't `ToSnake` already replace spaces with `_`? Am I'm missing something here?

Reply:
```go
package client

import (
	"testing"

	"github.com/cloudquery/plugin-sdk/caser"
)

func Test(t *testing.T) {
	t.Log(caser.New().ToSnake("Some long name"))
}
```

output: `some long name`

Reply:
OK so we should open a separate issue to fix this in the SDK, and follow up here once we fix the SDK caser

Reply:
https://github.com/cloudquery/plugin-sdk/issues/728